common/Field_Dimensions.cpp

```diff
@@ -9,4 +9,7 @@ const Field_Dimensions Field_Dimensions::Double_Field_Dimensions(
     1.000f, 0.500f, 1.000f, 0.500f, 10.400f, 7.400f);
 
 Field_Dimensions Field_Dimensions::Current_Dimensions =
-    Field_Dimensions::Double_Field_Dimensions;
\ No newline at end of file
+    Field_Dimensions::Double_Field_Dimensions;
+
+const Field_Dimensions Field_Dimensions::Default_Dimensions =
```

Comment:
I think that I could use this in the cases where I have to hardcode the values (I think that would be cleaner than adding a bunch of variables), would that be ok?